src/NinjaTools.FluentMockServer/Models/Verify.cs

```diff
@@ -1,9 +1,35 @@
 using System;
+using NinjaTools.FluentMockServer.Models.HttpEntities;
+using NinjaTools.FluentMockServer.Models.ValueTypes;
 
-namespace NinjaTools.FluentMockServer.Domain.Models.HttpEntities
+namespace NinjaTools.FluentMockServer.Models
 {
-    public partial class Verify : IEquatable<Verify>
+    /// <summary>
+    ///     Model used to describe what to verify.
+    /// </summary>
+    public class Verify : IEquatable<Verify>
     {
+        public Verify(HttpRequest httpRequest, VerificationTimes times)
+        {
+            Times = times;
+            HttpRequest = httpRequest;
+        }
+
+        /// <summary>
+        ///     The to be matched <see cref="HttpRequest" />.
+        /// </summary>
+        public HttpRequest HttpRequest { get;  }
+
+        /// <summary>
+        ///     How many <see cref="Times" /> the request is expected to have occured.
+        /// </summary>
+        public VerificationTimes Times { get; }
+
+        public static Verify Once(HttpRequest httpRequest = null)
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use the overloading mechanism instead of the optional parameters.](https://app.codacy.com/app/git_36/NinjaTools.FluentMockServer/pullRequest?prid=4722826)